android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotificationHelper.java

```diff
@@ -128,14 +143,53 @@ public void sendNotificationScheduledCore(Bundle bundle) {
         }
     }
 
-    public void sendNotification(Bundle bundle) {
+
+    public void sendNotification(final Bundle bundle) {
+        String imageUrl = bundle.getString("imageUrl");
+        if( imageUrl == null ){
+            sendNotificationWithImage( bundle, null );
+            return;
+        }
+        ImagePipeline imagePipeline = Fresco.getImagePipeline();
+
+        ImageRequest imageRequest = ImageRequestBuilder
+                .newBuilderWithSource( Uri.parse(imageUrl) )
+                .setRequestPriority(Priority.HIGH)
+                .setLowestPermittedRequestLevel(ImageRequest.RequestLevel.FULL_FETCH)
+                .build();
+        DataSource<CloseableReference<CloseableImage>> dataSource =
+                imagePipeline.fetchDecodedImage(imageRequest, mContext);
+
+
+        dataSource.subscribe(new BaseBitmapDataSubscriber() {
+            @Override
+            public void onNewResultImpl(@Nullable Bitmap bitmap) {
+                if (bitmap == null) {
+                    Log.d(TAG, "Bitmap data source returned success, but bitmap null.");
+                    sendNotificationWithImage(bundle, null);
```

Comment:
This line and the return statement are not needed.  You can just log the messages and continue.  I'd also consider looking a verbose level.